koku/koku/celery.py

```diff
@@ -112,6 +111,20 @@ def task(self, *args, **kwargs):
 # Collect prometheus metrics.
 app.conf.beat_schedule["db_metrics"] = {"task": "koku.metrics.collect_metrics", "schedule": crontab(minute="*/15")}
 
+
+# optionally specify the weekday and time you would like the clean volume task to run
+CLEAN_VOLUME_DAY_OF_WEEK = ENVIRONMENT.get_value("CLEAN_VOLUME_DAY_OF_WEEK", default="sunday")
+CLEAN_VOLUME_UTC_TIME = ENVIRONMENT.get_value("CLEAN_VOLUME_UTC_TIME", default="00:00")
+CLEAN_HOUR, CLEAN_MINUTE = CLEAN_VOLUME_UTC_TIME.split(":")
+if CLEAN_VOLUME_DAY_OF_WEEK:
+    clean_schedule = crontab(day_of_week=CLEAN_VOLUME_DAY_OF_WEEK, hour=int(CLEAN_HOUR), minute=int(CLEAN_MINUTE))
+else:
```

Comment:
You could probably just get rid of this else since `CLEAN_VOLUME_DAY_OF_WEEK` gets a default value assigned above. That should hopefully fix the codecov patch coverage problem too.